client/app/bundles/buttons/button.scss

```diff
@@ -0,0 +1,79 @@
+@import "base.scss";
+
+
+button:hover{
+         
+}
+.buttonFilledM{
+         fontSize: 18px;
+         color: #6D0839;
+         font-weight: bold;
+         background-color: #FFFFFF;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         padding-left: 15px;
+         padding-right: 15px;
+         width: 203px;
+         border-radius: 5px;
+         border-color: #FFFFFF;
+         border-width: 0px;
+         cursor: pointer;
+ }
+
+.buttonGhostM{
+         font-size: 18px;
+         color: #FFFFFF;
+         font-weight : bold;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         width: 128px;
+         border-radius: 5px;
+         opacity: 0.95;
```

Comment:
Also sorry for not commenting on this before -- let's define a background color to be a 95% opacity white instead of using the opacity property here!